### ISSUE TITLE
Can't get 1.0.0b5 into use

### ISSUE THREAD

**Issue by jlatvala** [py-smart-gardena/hass-gardena-smart-system#114]
I noticed the 0.2.5b4 didn't work anymore. 
-Installed 1.0.0b5 and had an error in log (about gardena authentication, don't remember details anymore).
-Opened https://developer.husqvarnagroup.cloud/ and my app was gone.
-Created new app in https://developer.husqvarnagroup.cloud/
-Redirect URLs is now mandatory field in https://developer.husqvarnagroup.cloud/ **<-- !!!** (I used "none")
-Opened Gardena Smart System integration's settings/configuration in HA's Integrations page.
-Tried to find a place to update new Application Key, couldn't find a way **<-- !!!**
-Deleted Gardena Smart System integration
-Added back Gardena Smart System integration 1.0.0b5 
-Gardena Smart System integration's UI asked email, passwd, Application Key
-I typed in email, password, Application Key, nothing happens, the same UI opens again and field are empty. **<-- !!!** (log attached below)
--> Gardena Smart System integration couldn't be taken into use **<-- !!!**

Tried 0.2.5b4  again. Same problem, typing email, password and Application Key doesn't work.

Log for 1.0.0b5:
```
This error originated from a custom integration.

Logger: custom_components.gardena_smart_system.config_flow
Source: custom_components/gardena_smart_system/config_flow.py:131
Integration: gardena_smart_system (documentation, issues)
First occurred: 08:23:59 (2 occurrences)
Last logged: 08:25:59

Unexpected exception
Traceback (most recent call last):
  File "/config/custom_components/gardena_smart_system/config_flow.py", line 66, in async_step_user
    await try_connection(
  File "/config/custom_components/gardena_smart_system/config_flow.py", line 131, in try_connection
    await smart_system.authenticate()
  File "/usr/local/lib/python3.9/site-packages/gardena/smart_system.py", line 70, in authenticate
    self.token = await self.client.fetch_token(
  File "/usr/local/lib/python3.9/site-packages/authlib/integrations/httpx_client/oauth2_client.py", line 131, in _fetch_token
    return self.parse_response_token(resp.json())
  File "/usr/local/lib/python3.9/site-packages/authlib/oauth2/client.py", line 380, in parse_response_token
    self.handle_error(error, description)
  File "/usr/local/lib/python3.9/site-packages/authlib/integrations/httpx_client/oauth2_client.py", line 80, in handle_error
    raise OAuthError(error_type, error_description)
authlib.integrations.base_client.errors.OAuthError: unauthorized_client: Unauthorized client: `grant_type` is invalid

```

**Comment by RobG80** [comment]
me too

**Comment by Korsbakke** [comment]
Identical issue. Husqvarna portal seems to have made some changes.

**Comment by grm** [comment]
TLDR : please update to 1.0.0b900

And from the other post : 

> ...
> I juste released a beta version of 1.0.0b900 which should include all new functionalities and use the new credentials information so you have to deconfigure and reconfigure the integration. I have still display bug during configuration to solve, but everything seems to work as intended.
> 
> Please tell me if you have any issue


I close this issue as it should be tracked in the other ticket.
